clients/osv/src/main/kotlin/OsvService.kt

```diff
@@ -46,6 +46,7 @@ import retrofit2.http.Path
 interface OsvService {
     companion object {
         const val BATCH_REQUEST_MAX_SIZE = 1000
+        const val DEFAULT_SERVER_URL = "https://api.osv.dev"
```

Comment:
Not sure, does it even make sense to allow other values than the default production or staging URLs because users could run their own instance of OSV? Because if not, also the OSV advisor should be changed to not allow arbitrary URLs.

Reply:
That's a good question. It seems like [the infrastructure behind OSV also is Open Source](https://github.com/google/osv.dev), so in the same way as [ClearlyDefined](https://github.com/clearlydefined/service) and [VulnerableCode](https://github.com/aboutcode-org/vulnerablecode) you could run your own OSV server. With that in mind arbitrary URLs could be needed.

Reply:
IMO it does not harm to allow arbitrary URLs, so there is almost zero win in introducing such limitation (in general, not just for OSV).

Reply:
@fviernau Would you like to keep the `Server` enum?

Reply:
> @fviernau Would you like to keep the `Server` enum?

I don't mind, to me constants are ok as well.

Reply:
Yes, I agree that'd be a good compromise: Remove the `Server` enum, but have

    const val PRODUCTION_SERVER_URL = "https://api.osv.dev"

and

    const val STAGING_SERVER_URL = "https://api-staging.osv.dev"

to not loose the information where the staging server is hosted.